src/main/java/com/hrznstudio/galacticraft/screen/PlayerInventoryGCScreenHandler.java

```diff
@@ -169,4 +170,41 @@ public int getMaxStackAmount() {
             return Pair.of(PlayerScreenHandler.BLOCK_ATLAS_TEXTURE, new Identifier(Constants.MOD_ID, Constants.SlotSprites.OXYGEN_TANK));
         }
     }
+
+    public ItemStack transferSlot(PlayerEntity player, int index) {
+        ItemStack stack = ItemStack.EMPTY;
+        Slot slotFrom = (Slot)this.slots.get(index);
```

Comment:
I assume the slots list contains `Slot`s so the cast is unnecessary.
```suggestion
        Slot slotFrom = this.slots.get(index);
```